Mianta/06_challenges/stocks_challenge.py

```diff
@@ -0,0 +1,60 @@
+from http import client
+
+
+print("Challenge 3.2: Playing with the stock market")
+
+print()
+
+# Creating variables to store the current (approximate) market price of these 5 companies - Amazon, Apple, Facebook, Google and Microsoft.
+
+amzn = 3000
+apple = 100
+fb = 250
+google = 1400
+msft = 200
+
+print("Challenge 3.2.1: Taking user input")
+# asked the client their name and saved it to the variable.
+client_name = input ('Please enter your name...-')
+if client_name.isdigit() == False:
```

Comment:
Here, you are checking whether the client input is a number, and if it is a number, you ask for input again. In other words, this should be ==True instead of False.

Reply:
if client_name.isdigit() == False: 
this should be 
if client_name.isdigit() == True:

Reply:
that is a way simpler what to write it then what I wrote Thank you!